src/components/views/sul-people/sul-people-table-view.tsx

```diff
@@ -95,34 +95,34 @@ const SulPeopleTableView = ({items, hasHeading}: Props) => {
         >
           Showing {displayedItems.length} of {items.length}
         </caption>
-        <Thead className="sr-only sm:not-sr-only">
-          <Tr className="block sm:hidden md:table-row">
+        <Thead className="sr-only lg:not-sr-only">
+          <Tr className="lock sm:hidden lg:!table-row">
```

Comment:
should this be `block` instead of `lock`?

Reply:
Ooo Great find. Yes it should be